src/main/java/org/mitre/inferno/FHIRPathEvaluator.java

```diff
@@ -0,0 +1,57 @@
+package org.mitre.inferno;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.hl7.fhir.exceptions.FHIRException;
+import org.hl7.fhir.r5.context.SimpleWorkerContext;
+import org.hl7.fhir.r5.formats.JsonParser;
+import org.hl7.fhir.r5.model.Base;
+import org.hl7.fhir.r5.model.DataType;
+import org.hl7.fhir.r5.model.Resource;
+import org.hl7.fhir.r5.utils.FHIRPathEngine;
+
+public class FHIRPathEvaluator extends FHIRPathEngine {
+
+  public FHIRPathEvaluator() throws IOException {
+    super(new SimpleWorkerContext());
+  }
+
+  @Override
+  public String evaluateToString(Base base, String path) {
```

Comment:
Do we think there's a cleaner way to create an array here, rather than doing three-lined string concatenation? Or are we good with this method?

Reply:
I wish I could just use `new Gson.toJson()` on a list containing the results, but this was the best I could come up with using the custom `baseToJson` function. Happy to take suggestions though!

Reply:
Wait I found [this cool guide](https://sites.google.com/site/gson/gson-user-guide#TOC-Custom-Serialization-and-Deserialization). It looks simple enough to try implementing a `JsonSerializer<Base>`, so I'll see what I can do

Reply:
It worked! ... but at what cost? Lol
```suggestion
private class Container {
    private String type;
    private Base value;

    private Container(String type, Base value) {
      this.type = type;
      this.value = value;
    }
  }

  private class BaseSerializer implements JsonSerializer<Base> {
    @Override
    public JsonElement serialize(Base item, Type typeOfSrc, JsonSerializationContext context) {
      return com.google.gson.JsonParser.parseString(baseToString(item));
    }
  }

  public FHIRPathEvaluator() throws IOException {
    super(new SimpleWorkerContext());
  }

  @Override
  public String evaluateToString(Base base, String path) {
    Gson gson = new GsonBuilder().registerTypeAdapter(Base.class, new BaseSerializer()).create();
    List<Container> result = this.evaluate(base, path)
        .stream()
        .map(item -> new Container(item.fhirType(), item))
        .collect(Collectors.toList());
    return gson.toJson(result);
  }
```